src/actions/assetsActions.js

```diff
@@ -468,11 +470,18 @@ export const fetchAllAccountsTotalBalancesAction = () => {
       return;
     }
 
+    const archanovaAccountAddress = accountsAddresses.find((address) => isArchanovaAccountAddress(address, accounts));
+
     try {
       await Promise.all(availableChainProtocols.map(async ({ chain, zapperProtocolIds, zapperNetworkId }) => {
+        // we don't need to pull multi chain balances for Archanova account, only Ethereum
+        const requestForAddresses = chain !== CHAIN.ETHEREUM
+          ? accountsAddresses.filter((address) => !addressesEqual(archanovaAccountAddress, address))
```

Comment:
Seems a bit simplier this way, and avoid some boiler plate:

```suggestion
          ? accountsAddresses.filter((address) => getAccountTypeByAddress(address, accounts) === ACCOUNT_TYPES.ETHERSPOT_SMART_WALLET)
```

Reply:
Good catch 👍